src/screenshots/capturer.js

```diff
@@ -86,25 +87,29 @@ export default class Capturer {
 
         this.testEntry.path = pathForReport;
 
-        await this._takeScreenshot(pageInfo, screenshotPath);
+        await this._takeScreenshot(screenshotPath, pageWidth, pageHeight);
 
         this.testEntry.hasScreenshots = true;
 
-        await generateThumbnail(screenshotPath);
+        // NOTE: generateThumbnail is not available on Linux yet. Keep tracking the https://github.com/DevExpress/testcafe-browser-tools/issues/12
+        if (!linux)
+            await generateThumbnail(screenshotPath);
 
         return screenshotPath;
     }
 
-    async captureError ({ pageInfo, screenshotRequired }) {
+    async captureError ({ screenshotRequired, pageWidth, pageHeight }) {
         if (!screenshotRequired || !this.enabled)
             return null;
 
         var fileName = this._getFileName(true);
         var { screenshotPath } = this._getSreenshotPath(joinPath('errors', fileName));
 
-        await this._takeScreenshot(pageInfo, screenshotPath);
+        await this._takeScreenshot(screenshotPath, pageWidth, pageHeight);
 
-        await generateThumbnail(screenshotPath);
+        // NOTE: generateThumbnail is not available on Linux yet. Keep tracking the https://github.com/DevExpress/testcafe-browser-tools/issues/12
```

Comment:
wow, I'm not yet tracking https://github.com/DevExpress/testcafe-browser-tools/issues/12, why d'you say to keep tracking it? Maybe `subscribe to keep track` as I wrote somewhere - that's a different story.

```
        // NOTE: generateThumbnail is not available on Linux yet. Subscribe to https://github.com/DevExpress/testcafe-browser-tools/issues/12 to keep track.
```